index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@700&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@200&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/e1cc58c746.js" crossorigin="anonymous"></script>
+</head>
+<body>
+    <section class="front_page">
+        <nav class="nav_bar">
+            <h2 class="logo">My Logo</h2>
+            <div class="menue_bar">
+                <div class="bar"></div>
+                <div class="bar"></div>
+                <div class="bar"></div>
+            </div>
+        </nav>
+        <div class="porto_wrapper">
+            <h1 class="headline">I'm Addisalem Glad to see you!</h1>
+            <p class="description">I am a software developer! I can help you build a product, feature or website look through some of my work and experiance! if you like what you see or have a project you need coded don't hesitate to contact me. </p>
+            <h2 class="connect">LETS CONNECT</h2>
```

Comment:
- [ ] This title "LETS CONNECT" should be inside `a` HTML tag due to you will use this link to go to another section in your portfolio 😉 For example:

```
<h2 class="connect"><a>LETS CONNECT</a></h2>
```

- [ ]  Probably your design will change after adding the previous requirement, please fix your styles to make your design similar to the Figma design you selected if this situation happens.